beacon-chain/p2p/service.go

```diff
@@ -348,13 +349,14 @@ func (s *Service) MetadataSeq() uint64 {
 // RefreshENR uses an epoch to refresh the enr entry for our node
 // with the tracked committee id's for the epoch, allowing our node
 // to be dynamically discoverable by others given our tracked committee id's.
-func (s *Service) RefreshENR(epoch uint64) {
+func (s *Service) RefreshENR(slot uint64) {
 	// return early if discv5 isnt running
 	if s.dv5Listener == nil {
 		return
 	}
 	bitV := bitfield.NewBitvector64()
-	committees := cache.CommitteeIDs.GetIDs(epoch)
+	committees := sliceutil.UnionUint64(cache.CommitteeIDs.GetAttesterCommitteeIDs(slot),
```

Comment:
this isnt accurate, we still refresh by epoch. What we need to do here is loop through all the slots in the epoch and aggregate the relevant committe indexes